docs/resources/covid-19.md

```diff
@@ -1,44 +1,182 @@
 # Covid-19 and Travel Safety 
 
-Access travel Restriction data for over 200 countries
 
-[Keep Travelers informed with Amadeus Travel Restriction API ](https://developers.amadeus.com/blog/-introducing-amadeus-covid-19-travel-restrictions-api)
 
-Powered by [Riskline](https://riskline.com/), the travel restrictions data is updated every hour to so you can keep your travelers informed of the latest developments on their trip. 
+"Can I travel to South Korea? Thailand? or Costa Rica? Do I need to have a PCR test? before the flight? after I arrive? How many days of quarantines are required? Where can I find this information?" 
+
+Don't worry, with Amadeus Self Service APIs, you will be able to have the details information of **current restrictions, Number of covid-19 cases, Entry requirements, and Safety scores, etc** with more details in a searched destinations with one single API.
+
+and even more, **safety information** can be retrieved for the destination you would like to visit so that you will be aware of where you are heading to. 
 
 ## Search by an area
-the [Travel Restrictions API](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/travel-restrictions) can be searched by a country, city or region. 
- 
+
+There are 2 APIs to achieve the information 
+1. [Amadeus Travel Restrictions API](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/travel-restrictions) for covid-19 related information 
+2. [Amadeus Safety Place API](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/safe-place) for Safety information
+
+[Travel Restrictions API](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/travel-restrictions) can be searched **by a country, city, or region**. 
+
 ```bash
-curl https://test.api.amadeus.com/v1/duty-of-care/diseases/covid19-area-report?countryCode=KR&cityCode=SEL
+curl https://test.api.amadeus.com/v1/duty-of-care/diseases/covid19-area-report?countryCode=FR&cityCode=PAR
 ```
-
 !!! information
     Country code(2 letters code) is defined in [ISO 3166](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) and city code(3 letters code) is defined in [IATA](https://www.iata.org/en/publications/directories/code-search/), which you can also use [Amadeus Airport & City Search API](https://developers.amadeus.com/self-service/category/air) to get.
```

Comment:
Country code -> The country code
I find this part a bit hard to understand, maybe split in 2 sentences?
suggestion: "You can use the [Airport & City Search API](https://developers.amadeus.com/self-service/category/air) to get this information". 
I also think the word "Amadeus" is being used a lot before each API but I don't think it's necessary. It makes sense maybe in a blog article, but in a documentation not really.